airbyte-bootloader/src/test/java/io/airbyte/bootloader/ProtocolVersionCheckerTest.java

```diff
@@ -199,13 +213,17 @@ void testValidateAllConnectorsAreUpgraded(final boolean supportAutoUpgrade) thro
         Map.entry(dest2, V1_0_0),
         Map.entry(dest3, V2_0_0)));
 
+    final ProtocolVersionChecker protocolVersionChecker =
+        new ProtocolVersionChecker(jobPersistence, expectedVersionRange, configRepository, Optional.of(definitionsProvider));
```

Comment:
nit: I find targetVersionRange clearer because it is the input of the checker, not just the expected result.